integrations/scheduler/client.go

```diff
@@ -1,30 +0,0 @@
-package scheduler
```

Comment:
Could we keep the scheduler for now as is and not delete in this PR?

Reply:
got it back, but renamed to `_scheduler`, as I don't want to fix it to connection vars as I'm not sure what we'll do there.